Broken syntax highlighting for vlang

### Summary

This is the current rendering in Helix (v24.7)

![Image](https://github.com/user-attachments/assets/dd099296-a0f0-47ea-bec1-7cd9e2f75498)

And this is Neovim:

![Image](https://github.com/user-attachments/assets/7375e7ff-de07-4699-ad71-2f224c037903)

Both uses treesitter for their syntax.

### Reproduction Steps

```vlang
struct ManifestBlob {
        media_type string @[json: 'mediaType']
        size       int
        digest     string
}
```

### Helix log

```
Language             LSP                  DAP                  Formatter            Highlight            Textobject           Indent               

v                    ✓ v-analyzer         None                 None                 ✓                    ✓                    ✓                    
```


```
2025-02-16T16:45:48.891 helix_lsp::transport [ERROR] vlang-language-server err: <- StreamClosed
2025-02-16T16:45:48.891 helix_lsp::client [WARN] language server failed to terminate gracefully - server closed the stream
2025-02-16T16:45:48.892 helix_lsp::transport [ERROR] vlang-language-server err: <- IO(Os { code: 32, kind: BrokenPipe, message: "Broken pipe" })
```

### Platform

Linux

### Terminal Emulator

Foot

### Installation Method

source

### Helix Version

helix 24.7 (3f528493)

--- a/languages.toml
+++ b/languages.toml
@@ -2406,7 +2406,7 @@ indent = { tab-width = 4, unit = "\t" }
 
 [[grammar]]
 name = "v"
-source = {git = "https://github.com/vlang/v-analyzer", subpath = "tree_sitter_v", rev = "12ed90d735051ad961c54c4f0ec126354db4f966"}
+source = {git = "https://github.com/vlang/v-analyzer", subpath = "tree_sitter_v", rev = "59a8889d84a293d7c0366d14c8dbb0eec24fe889"}
 
 [[language]]
 name = "verilog"

--- a/languages.toml
+++ b/languages.toml
@@ -2406,7 +2406,7 @@ indent = { tab-width = 4, unit = "\t" }
 
 [[grammar]]
 name = "v"
-source = {git = "https://github.com/vlang/v-analyzer", subpath = "tree_sitter_v", rev = "e14fdf6e661b10edccc744102e4ccf0b187aa8ad"}
+source = {git = "https://github.com/vlang/v-analyzer", subpath = "tree_sitter_v", rev = "12ed90d735051ad961c54c4f0ec126354db4f966"}
 
 [[language]]
 name = "verilog"

--- a/runtime/queries/v/highlights.scm
+++ b/runtime/queries/v/highlights.scm
@@ -1,4 +1,4 @@
-(comment) @comment
+[(line_comment) (block_comment)] @comment
 
 (module_clause
  (identifier) @namespace)
@@ -84,9 +84,9 @@
 ] @string
 
 (string_interpolation
- (braced_interpolation_opening) @punctuation.bracket
- (interpolated_expression) @embedded
- (braced_interpolation_closing) @punctuation.bracket)
+ (interpolation_opening) @punctuation.bracket
+ (interpolation_expression) @embedded
+ (interpolation_closing) @punctuation.bracket)
 
 (attribute) @attribute
 

--- a/runtime/queries/v/injections.scm
+++ b/runtime/queries/v/injections.scm
@@ -1,4 +1,4 @@
-((comment) @injection.content
+([(line_comment) (block_comment)] @injection.content
  (#set! injection.language "comment"))
 
 ((sql_expression) @injection.content

--- a/runtime/queries/v/textobjects.scm
+++ b/runtime/queries/v/textobjects.scm
@@ -22,6 +22,6 @@
 (struct_field_declaration
   ((_) @parameter.inside) @parameter.around)
 
-(comment) @comment.inside
-(comment)+ @comment.around
+[(line_comment) (block_comment)] @comment.inside
+[(line_comment)+ (block_comment)+] @comment.around
 

